src/components/pages/contact/leaflet/bring-home.tsx

```diff
@@ -1,13 +1,14 @@
 import styled from 'styled-components';
 import { theme } from '../../../layout/theme';
 import React, { useCallback, useEffect, useState } from 'react';
+import { useMap } from 'react-leaflet';
 
 const BringMeHomeStyled = styled.a`
   position: absolute;
   left: 50%;
   bottom: 5px;
   transform: translate(-50%);
-  z-index: 1;
+  z-index: 400;
```

Comment:
Why `400`? Wouldn't `2` be enough?

Reply:
No, I tried even 390 and it did not work out. But 400 worked.